core/services/gateway/handlers/workflow/handler.go

```diff
@@ -0,0 +1,91 @@
+package workflow
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/api"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/config"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers"
+)
+
+const (
+	MethodAddWorkflow = "add_workflow"
+	// NOTE: more methods will go here: CRUD for workflow specs; HTTP trigger/action/target; etc.
```

Comment:
More of a TODO than a NOTE probably

Reply:
This TODO/NOTE is not correct anyway (I left that comment on Jin's PR too). We will not reuse this handler for workflow spec CRUDs.